lib/gui/pages/settings/templates/settings.tpl.html

```diff
@@ -46,7 +46,11 @@ <h3 class="subtitle">Advanced</h3>
     <label>
       <input type="checkbox"
         ng-model="settings.currentData.unsafeMode"
-        ng-change="settings.enableDangerousSetting('unsafeMode', 'Are you sure you want to turn this on? You will be able to burn to your system drives.')">
+        ng-change="settings.enableDangerousSetting('unsafeMode', {
+          message: 'Are you sure you want to turn this on? You will be able to burn to your system drives.',
```

Comment:
"overwrite" or "destroy [data on]" may be clearer, Also the phrase "if you are not careful" at the end might make it clearer.